instrumentation/net/http/otelhttp/filters/header.go

```diff
@@ -12,8 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !go1.14
```

Comment:
Shouldn't this be the file to be deleted and the other retained?  This was included to support <1.14 and the other file uses the `Values()` method that was added in 1.14 and should be available in all presently supported versions.

Reply:
From my understanding of build constraints including a `!` means the matching constrains will not be built for. Thus this file is the one built for all non Go 1.14 build (i.e. this was the one supported for Go 1.15). This seems to be inline with what the file name is (i.e. `*_nongo14.go`). The other file deleted, `header_go14.go` had the build constraint (`+build go1.14`) that included it in builds for Go 1.14 AFAIK.

Reply:
I think @Aneurysm9 is correct.  This file was the pre 1.14 version.  Now we can retrieve this slice via the `Values()` method.  I think we should switch to the other file, rename it to just header.go and remove the build constraint.